src/batched_inference_executor.py

```diff
@@ -378,22 +377,22 @@ def run(self):
     parser = argparse.ArgumentParser(allow_abbrev=False)
 
     # Required params
-    parser.add_argument("--device-id", type=int, required=True)
-    parser.add_argument("--model-type", type=str, required=True)
-    parser.add_argument("--model", type=str, required=True)
-    parser.add_argument("--batch-size", type=int, required=True)
-    parser.add_argument("--distribution-type", type=str, required=True)
-    parser.add_argument("--rps", type=float, required=True)
-    parser.add_argument("--tid", type=int, required=True)
-    parser.add_argument("--kafka-server", type=str, required=True)
-    parser.add_argument("--run-id", type=str, required=True)
+    parser.add_argument("--device-id", type=int, default=0)
+    parser.add_argument("--model-type", type=str, default='vision')
+    parser.add_argument("--model", type=str, default='vgg19')
+    parser.add_argument("--batch-size", type=int, default=32)
+    parser.add_argument("--distribution-type", type=str, default='poisson')
+    parser.add_argument("--rps", type=float, default=5000)
+    parser.add_argument("--tid", type=int, default=1)
+    parser.add_argument("--kafka-server", type=str, default="10.52.3.75:29092")
+    parser.add_argument("--run-id", type=str, default='cuac')
```

Comment:
Can these all be required args?

Reply:
Sure, this is how it was prior to my changes but I'll make it required.

Reply:
Done